src/Backends/Banshee.GStreamer/Banshee.GStreamer/PlayerEngine.cs

```diff
@@ -209,9 +209,11 @@ protected override void Initialize ()
 
         public override void Dispose ()
         {
+            bp_stop (handle, true);
```

Comment:
It's called from `PlayerEngine.Close`. I had the deinit call previously somewhere else where it didn't work without `bp_stop`.
I'll remove the call.